package.json

```diff
@@ -58,6 +58,7 @@
     "pino-pretty": "^9.1.1",
     "postgres-migrations": "^5.3.0",
     "react": "17.0.2",
+    "react-bootstrap": "^2.6.0",
```

Comment:
not sure how i feel about using react-bootstrap. Its such a heavy library, and this app is kinda heavy to begin with

Reply:
why do you need it?